packages/openapi3/test/tsp-openapi3/test-host.ts

```diff
@@ -0,0 +1,46 @@
+import { createTestWrapper } from "@typespec/compiler/testing";
+import assert from "node:assert";
+import { convertOpenAPI3Document } from "../../src/index.js";
+import { OpenAPI3Document, OpenAPI3Schema, Refable } from "../../src/types.js";
+import { createOpenAPITestHost } from "../test-host.js";
+
+export async function createConverterTestRunner() {
```

Comment:
probably wouldn't export that here as this is just needed for this sanity check

Reply:
Removed the runner!